File: individual/append_listNode.cpp
ListNode* append(ListNode* &node, int value, bool& flag){
    
    ListNode* Newnode = new ListNode(value);
    if (Newnode == nullptr){
        flag = true;
        return node;
    }
    if (node == nullptr){
        return Newnode;
    }
    ListNode* temp = node;
    while (temp->next != nullptr){
        temp = temp->next;
    }
    temp->next = Newnode;

    return node;
}

File: individual/append_listNode.py
def append(node, value):
    new_node = ListNode()
    new_node.next = None
    new_node.val = value

    if node is None:
        return new_node
    temp = node
    while temp.next is not None:
        temp = temp.next
    temp.next = new_node
    return node
